packages/theme-light-extension/style/index.css

```diff
@@ -16,3 +16,14 @@ pre {
   font-size: var(--jp-code-font-size);
   line-height: var(--jp-code-line-height);
 }
+
+/* A special class to be used to make elements needed for screenreaders
+ * invisible to users who do not use screen readers */
+.screenReaderOnly {
```

Comment:
I think this is redundant with the `.screenReaderOnly` class above.

Reply:
The .screenReaderOnly class above is for the dark extension, whereas this is the light extension. Perhaps we need to create an accessibility-related style sheet that gets `@import`ed in all themes to keep this in one place??